docs/SUMMARY.md

```diff
@@ -26,9 +26,9 @@
         - [Metrics Ping](user/pings/metrics.md)
         - [Events Ping](user/pings/events.md)
         - [Custom Pings](user/pings/custom.md)
-- [Development](dev/index.md)
+- [Developing the Glean SDK](dev/index.md)
     - [Testing](dev/testing.md)
-    - [Android](dev/android/index.md)
+    - [Android Component](dev/android/index.md)
```

Comment:
nit: mh, this recalls too much "Android Components" :-) Would "Android Bindings" or "Android APIs"?